commands/apps.go

```diff
@@ -328,12 +329,26 @@ func RunAppsListDeployments(c *CmdConfig) error {
 
 // RunAppsGetLogs gets app logs for a given component.
 func RunAppsGetLogs(c *CmdConfig) error {
-	if len(c.Args) < 3 {
+	if len(c.Args) < 1 {
 		return doctl.NewMissingArgsErr(c.NS)
 	}
 	appID := c.Args[0]
-	deploymentID := c.Args[1]
-	component := c.Args[2]
+	component := "web"
```

Comment:
Should we default this to the first component in the active deployment instead?

Reply:
Yes, that would be better.